bookkeeper-server/src/test/java/org/apache/bookkeeper/replication/TestReplicationWorker.java

```diff
@@ -567,15 +567,12 @@ public void testRWZKSessionLost() throws Exception {
                 Thread.sleep(1000);
             }
             assertTrue("Replication worker should be running", rw.isRunning());
+
             stopZKCluster();
+            Thread.sleep(1000);
```

Comment:
Instead of sleep, can we check something like '!zk.getState().isConnected()' in a loop